src/main/java/org/jabref/gui/icon/IconTheme.java

```diff
@@ -192,71 +192,71 @@ public static URL getIconUrl(String name) {
         RANK3(MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR_OUTLINE, MaterialDesignIcon.STAR_OUTLINE),
         RANK4(MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR_OUTLINE),
         RANK5(MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR, MaterialDesignIcon.STAR),
-        WWW(MaterialDesignIcon.WEB) /*css: web*/,
```

Comment:
I fear removing this comments is a bad idea

Reply:
The comments in this source file were the always (!) the same as the reference to the icon. I see no reason why we should duplicate constant names.

Reply:
In case you can explain to me the sence of these comments, I can re-add them. All of them formatted the same way. I have the regexp ready.

I sat side-by-side by @matthiasgeiger when he worked on the icons at [JabCon](http://jabcon.jabref.org/) 2017. I think, they had to be used at that time when constants in the library were missing (and we had to adress the icon names by numbers (!)). Today, I see no reason to have them here.